main.go

```diff
@@ -794,18 +761,12 @@ func buildAndRun(pkgName string, config *compileopts.Config, stdout io.Writer, c
 				"runtime": runtimeGlobals,
 			}
 		}
-	} else if config.EmulatorName() == "wasmtime" {
-		// Wasmtime needs some special flags to pass environment variables
-		// and allow reading from the current directory.
-		args = append(args, "--dir=.")
+	} else if strings.Contains(config.Target.Emulator, "wazero") {
+		// wazero needs some special flags to pass environment variables
```

Comment:
maybe add the why?

```suggestion
		// wasm is sandboxed by default, so we explicitly propagate ENV variables used in tests.
```

We could also in the CLI allow env propagation, either opt-in or by default.. I wouldn't change the go library default, but for example, the CLI uses real time also which isn't a default.

Reply:
counterpoint is docker doesn't automatically propagate env from the host, so in general I prefer the explicitness.

Reply:
Yeah would probably be good to have a flag to opt-in the environment, that would allow tinygo test and go test to behave the same regardless of target, and similar use cases.

Reply:
for this though I would say that since environmentVars are already defined in tinygo we don't need to block on an auto-propagation thing. I also noticed docker doesn't do it, and it is likely not to have a lot of maintenance here in go, as I don't expect them to add host env sensitive variables into tests that often.